test/local_example.sh

```diff
@@ -98,5 +98,5 @@ mysql --table < ../common/select_customer80-_data.sql
 
 ./306_down_shard_0.sh
 
-./401_teardown.sh
```

Comment:
We should add the new backup steps here.

Reply:
done with the changes